### ISSUE TITLE
[Bug] Can not get balance status

### ISSUE THREAD

**Issue by dtb11288** [gcarq/freqtrade#402]
* Python Version: 3.6
  * Branch: Develop
  * Last Commit ID: [6cafa9120c5aa35599a5fdbb76fb994e7910b24c](https://github.com/gcarq/freqtrade/commit/6cafa9120c5aa35599a5fdbb76fb994e7910b24c)
 
## Describe the problem:
*Can not receive result with command /balance*

### Steps to reproduce:

  1. Just use the clean setup and start
  
### Observed Results:

  * What happened? Nothing except error !!!

### Relevant code exceptions or logs:

  ```
2018-01-18 14:45:32,181 - freqtrade.rpc.telegram - INFO - Executing handler: _balance for chat_id: 507292729
2018-01-18 14:45:49,079 - freqtrade.rpc.telegram - ERROR - Exception occurred within Telegram module
Traceback (most recent call last):
  File "/home/binh/freqtrade/freqtrade/rpc/telegram.py", line 108, in wrapper
    return command_handler(*args, **kwargs)
  File "/home/binh/freqtrade/freqtrade/rpc/telegram.py", line 391, in _balance
    currency["Rate"] = exchange.get_ticker('BTC_' + coin, False)['bid']
  File "/home/binh/freqtrade/freqtrade/exchange/__init__.py", line 138, in get_ticker
    return _API.get_ticker(pair, refresh)
  File "/home/binh/freqtrade/freqtrade/exchange/bittrex.py", line 124, in get_ticker
    pair=pair))
freqtrade.OperationalException: INVALID_MARKET params=(BTC_USDT)
  ```

**Comment by jblestang** [comment]
The BTC-USDT pair does not exists, this is a configuration issue.

**Comment by dtb11288** [comment]
there is no BTC_USDT in config or in any line of code either

**Comment by jblestang** [comment]
Your problem arises when you setup USDT as the base currency. Did you switch to USDT in your config file recently?

**Comment by shusso** [comment]
closing as this was fixed in #453
